src/components/Popup/Popup.js

```diff
@@ -1,103 +1,39 @@
 import { useEffect, useState } from "react";
 import { useKey } from "../../hooks/useKey";
-import PopupWithForm from "../PopupWithForm/PopupWithForm";
+
 import { useNavigate } from "react-router-dom";
+import { authorize, register } from "../../utils/auth";
+import Register from "../Register/Register";
+import Login from "../Login/Login";
+import FormDataProvider from "../../context/form-context";
 
-const Popup = ({ isOpen, onClose, popupType, changeState, changeStatus }) => {
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
-  const [username, setUsername] = useState("");
+const Popup = ({
+  isOpen,
+  onClose,
+  popupType,
+  changeState,
+  popupChangeType,
+  openTooltip,
+}) => {
   const navigate = useNavigate;
```

Comment:
please, delete unsed variables from your code

Reply:
Removed. along with other unused variables in the code.